### PR TITLE
Fixed broken storage driver link

### PR DESCRIPTION
**Pull request by mistyhacks**
Signed-off-by: Misty Stanley-Jones <misty@docker.com>

### CODE REVIEW

**Comment by stevvooe** [comment]
LGTM

**Comment by mistyhacks** [comment]
@stevvooe PTAL, I don't understand the tests.

File: docs/spec/index.md [hunk 1]
```diff
@@ -7,6 +7,6 @@ keywords: registry, service, images, repository,  json
 # Docker Registry Reference
 
 * [HTTP API V2](api.md)
-* [Storage Driver](../storage-drivers/index.md)
+* [Storage Driver](https://docs.docker.com/registry/storage-drivers/index.md)
```
Comment (dmcgowan) [inline docs/spec/index.md]: Does this link get cleaned up later when publishing? This URL does not work as is, https://docs.docker.com/registry/storage-drivers/ does

**Comment by dmcgowan** [comment]
@mistyhacks ignore codecov, for some reason we aren't able to turn it off :laughing:

**Comment by mistyhacks** [comment]
Yes, we use a bit of Javascript to clean it up. Linking to the .md files means that the links also work when browsing our source on Github. In this case, it probably should be the "cleaned up" version of the link since it's  at an absolute URL, so I'll clean it up.

**Comment by mistyhacks** [comment]
@dmcgowan PTAL again

**Comment by dmcgowan** [comment]
LGTM

**Pull request merged**